sdk/tables/azure-tables/src/utils/bufferSerializer.browser.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export function isUint8Array(value: any): boolean {
```

Comment:
Just wondering - It looks like the implementation of `isUint8Array` is the same between the browser and node.  Could we just do the `instanceof` check directly instead of introducing this helper and having two implementations?